### PR TITLE
Fix TableArray serialization bug

### LINKED ISSUE #72: Table arrays containing primitive arrays unnecessarily serialize inline.
The following TOML file:

```toml
[[table_array]]
primitive_list = [4, 5, 6]
```

When run through the functions Toml.ToModel & Toml.FromModel as follows:

```csharp

var toml = "[[table_array]]\nprimitive_list = [4, 5, 6]";

var model = Toml.ToModel(toml);
var tomlOut = Toml.FromModel(model);
```

Results in the following output:
```
table_array = [{primitive_list = [4, 5, 6]}]
```

I believe this may be a bug, as it appears that the serialization code otherwise prefers to use the non-inline style wherever possible.

### PR DESCRIPTION
**Pull request by youliao**
Fix #72

### CODE REVIEW

File: src/Tomlyn.Tests/BasicTests.cs [hunk 1]
```diff
@@ -1,18 +1,20 @@
 // Copyright (c) Alexandre Mutel. All rights reserved.
-// Licensed under the BSD-Clause 2 license. 
+// Licensed under the BSD-Clause 2 license.
 // See license.txt file in the project root for full license information.
 
 using System;
 using System.Collections.Generic;
+using System.Reflection;
 using System.Runtime.Serialization;
+using Newtonsoft.Json.Linq;
```
Comment (xoofx) [inline src/Tomlyn.Tests/BasicTests.cs:9]: Probably not needed?

File: src/Tomlyn.Tests/BasicTests.cs [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
 using NUnit.Framework;
 using Tomlyn.Model;
 
 namespace Tomlyn.Tests
 {
     public class BasicTests
     {
-        [Test]
+        [Test]
```
Comment (xoofx) [inline src/Tomlyn.Tests/BasicTests.cs:17]: Not sure what are the changes here? It makes the PR a bit noisy.

Comment (youliao) [reply src/Tomlyn.Tests/BasicTests.cs]: sorry，it's been revised.

**Comment by xoofx** [comment]
Seems that the PR is introducing a regression.

**Comment by xoofx** [comment]
Fixed it differently for now ffcc734

**Pull request closed without merging**